code/game/machinery/machinery.dm

```diff
@@ -366,7 +366,7 @@ Class Procs:
 		return 1
 	return 0
 
-/obj/machinery/proc/default_unfasten_wrench(mob/user, obj/item/weapon/wrench/W, time = 20)
+/obj/proc/default_unfasten_wrench(mob/user, obj/item/weapon/wrench/W, time = 20)

```

Comment:
this should probably be in object procs somewhere, instead of in machinery.dm if it's not machinery only now